src/ontology/config/mappings-to-uberon-bridge.rules

```diff
@@ -0,0 +1,16 @@
+prefix UBERON:    <http://purl.obolibrary.org/obo/UBERON_>
+prefix CL:    <http://purl.obolibrary.org/obo/CL_>
+
+# Make sure UBERON/CL classes, and only UBERON/CL classes, are on the object side
+subject==UBERON:* -> invert();
+!object==UBERON:* -> stop();
+
+# Ignore any mapping to/from an inexistent or obsolete class.
+predicate==* -> check_object_existence();
+predicate==* -> check_subject_existence();
+
+# Ignore any mapping where the same foreign term is mapped to more than one UBERON class.
+!cardinality==*:1 -> stop();
+
+# Create the hasDbXref annotation and annotate it with a subset of the mapping metadata
+object==UBERON:* && predicate==semapv:crossSpeciesExactMatch -> create_axiom('%subject_id SubClassOf: %object_id', "direct:metadata");
```

Comment:
Presumably you want those axioms to be created for both Uberon and CL classes?

Then you could either do:

```
(object==UBERON:* || object==CL:*) && predicate==semapv:crossSpeciesExactMatch -> ...;
```

But, since you already dropped all mappings with an object that is not in Uberon or CL above, you no longer need to filter on object and you could simply do:

```
predicate==semapv:crossSpeciesExactMatch -> ...;
```

Reply:
Awesome thank you!